src/Administration/Resources/app/administration/src/module/sw-login/view/sw-login-login/sw-login-login.html.twig

```diff
@@ -5,14 +5,14 @@
     @submit.prevent="loginUserWithPassword"
 >
     <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
-    {% block sw_login_login_form_headline %}
+        {% block sw_login_login_form_headline %}
```

Comment:
Hmm... The indentation of the Twig blocks is inconsistent and affects readability.